cli/command/service/list.go

```diff
@@ -116,7 +118,7 @@ func printTable(out io.Writer, services []swarm.Service, running map[string]int)
 		if service.Spec.Mode.Replicated != nil && service.Spec.Mode.Replicated.Replicas != nil {
 			replicas = fmt.Sprintf("%d/%d", running[service.ID], *service.Spec.Mode.Replicated.Replicas)
 		} else if service.Spec.Mode.Global != nil {
-			replicas = "global"
+			replicas = fmt.Sprintf("%d/%d global", running[service.ID], tasksPerService[service.ID])
```

Comment:
Since 1.13.0, task of global service will only on node which satisfies constraints. So this PR counts node which only has the corresponding task. PTAL @thaJeztah